tools/depends/target/libdcadec/libdcadec_android.patch

```diff
@@ -0,0 +1,14 @@
+diff --git a/libdcadec/dca_stream.c b/libdcadec/dca_stream.c
+index 156f874..0eccf6c 100644
+--- a/libdcadec/dca_stream.c
++++ b/libdcadec/dca_stream.c
+@@ -287,7 +287,7 @@ static int read_frame(struct dcadec_stream *stream, uint32_t *sync_p)
+         && sync != SYNC_WORD_EXSS_LE
+         && sync != SYNC_WORD_CORE_LE14
+         && sync != SYNC_WORD_CORE_BE14) {
+-        int c = DCA_FGETC(stream->fp);
++        int c = fgetc(stream->fp);
```

Comment:
Might be cleaner to alter the definition of DCA_FGETC rather than replacing it.
If DCA_FGETC  gets reused in further versions, we'd fail again...